src/sqlfluff/rules/L003.py

```diff
@@ -127,7 +127,7 @@ def _process_raw_stack(cls, raw_stack, tab_space_size=4):
                         # an increase in indentation? Can't quite
                         # remember the logic here. Let's go with that.
                         clean_indent = True
-                else:
+                elif not elem.is_type("placeholder"):
```

Comment:
L003 computes the desired indentation based on the first code on a line. Need to skip placeholders (template code) so it finds the first **SQL** code on the line.